src/components/radio/radio.ts

```diff
@@ -53,9 +71,13 @@ export class MdRadioGroup implements AfterContentInit {
   /** The currently selected radio button. Should match value. */
   private _selected: MdRadioButton = null;
 
+  private _changeSubscription: {unsubscribe: () => any} = null;
+
+  onTouched: () => any = () => {};
+
   /** Event emitted when the group value changes. */
   @Output()
-  change: EventEmitter<MdRadioChange> = new EventEmitter();
+  change: EventEmitter<any> = new EventEmitter();
```

Comment:
I changed the type of the event from the MdRadioChange to just the value of the radio button because if I found that if I didn't the ngModel I would bind to would get an MdRadioChange event as it's value instead of the intended value.  The changes related to this line up with similar code that I saw in the MdCheckbox implementation which I used as a reference for most of my changes.